AC 2.4.1 updates (#1293)
astronomer/docs: software/ac-support-policy.md

> For customers looking for less frequent upgrades and functional changes, we recommend following the LTS release channel exclusively.

Reviewer: ```suggestion
For customers looking for less frequent upgrades and functional changes, Astronomer recommends following the LTS release channel exclusively.
```

> For complex use cases that require customizing AC base image, read [Customize your Airflow image on Astronomer](customize-image.md).

Reviewer: ```suggestion
For complex use cases that require customizing AC base image, see [Customize your Airflow image on Astronomer](customize-image.md).
```

## Backport policy for bug and security fixes

> If a major stability bug in Astronomer Certified is identified by Astronomer, a fix will be backported to all LTS versions and only the latest stable version. For users on a stable version that is not latest, our team will recommend that you upgrade. Major issues in this category may result in significant delays in task scheduling as well as potential data loss.

Reviewer: ```suggestion
If a major stability bug in Astronomer Certified is identified by Astronomer, a fix is backported to all LTS versions and only the latest stable version. For users on a stable version that is not latest, Astronomer support will recommend that you upgrade. Major issues in this category may result in significant delays in task scheduling as well as potential data loss.
```

If a major security issue is identified, a fix will be backported and made available as a new AC hotfix version for _all_ available stable and LTS releases. Major issues in this category are classified by a combination of impact and exploitability.

> In rare instances, the Astronomer team may make an exception and backport a bug or security fix to a release that is beyond the commitment stated above. To submit a request for consideration, please reach out to your customer success manager.

Reviewer: ```suggestion
In rare instances, Astronomer support might make an exception and backport a bug or security fix to a release that is beyond the stated commitment. To submit a request for consideration, contact your customer success manager.
```

:::

### End of maintenance date

> Maintenance is discontinued the last day of the month for a given version. For example, if the maintenance window for a version of Astronomer Certified is January - June of a given year, that version will be maintained by Astronomer until the last day of June.

Reviewer: ```suggestion
Maintenance is discontinued on the last day of the month for a given version. For example, if the maintenance window for a version of Astronomer Certified is January - June of a given year, that version is maintained by Astronomer until the last day of June.
```

## Astronomer Certified lifecycle schedule

<!--- Version-specific -->

> The following tables contain the exact lifecycle for each published version of Astronomer Certified. These timelines are based on the LTS and Stable release channel maintenance policies.

Reviewer: ```suggestion
The following tables list the lifecycles for each published version of Astronomer Certified. These timelines are based on the LTS and Stable release channel maintenance policies.
```

| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |
| [2.4](https://github.com/astronomer/ap-airflow/blob/master/2.4.1/CHANGELOG.md)       | September 29, 2022 | March 2023            |

### LTS releases

| AC Version                                                                           | Release Date   | End of Maintenance Date |
| ------------------------------------------------------------------------------------ | -------------- | ----------------------- |
| [1.10.15](https://github.com/astronomer/ap-airflow/blob/master/1.10.15/CHANGELOG.md) | March 19, 2021 | September 2022          |
| [2.1](https://github.com/astronomer/ap-airflow/blob/master/2.1.4/CHANGELOG.md)       | May 21, 2021   | November 2022           |
| [2.3](https://github.com/astronomer/ap-airflow/blob/master/2.3.0/CHANGELOG.md)       | April 30, 2022 | October 2023            |

> If you have any questions or concerns, reach out to [Astronomer support](https://support.astronomer.io).

Reviewer: ```suggestion
If you have any questions or concerns, contact [Astronomer support](https://support.astronomer.io).
```
